ioctl/cmd/action/action.go

```diff
@@ -272,25 +241,93 @@ func sendAction(elp action.Envelope, signer string) error {
 			return nil
 		}
 	}
-	return sendRaw(selp)
+	return SendRaw(selp)
 }
 
-func isBalanceEnough(address string, act action.SealedEnvelope) (err error) {
+// Execute sends signed execution transaction to blockchain
+func Execute(contract string, amount *big.Int, bytecode []byte) error {
+	gasPriceRau, err := gasPriceInRau()
+	if err != nil {
+		return output.NewError(0, "failed to get gas price", err)
```

Comment:
ditto

Reply:
Using `0` when we don't hope previous error code to be overrode.
Here, the new error's code is got from `gasPrinceInRau`.